Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG NGINX_IMAGE_TAG=stable-alpine
 
 FROM debian:10-slim AS builder
 
-ARG HUGO_VERSION=0.62.2
+ARG HUGO_VERSION=0.57.2
```

Comment:
You are reverting my change. It should be 0.62.2

Reply:
done.